lib/spack/spack/installer.py

```diff
@@ -2148,20 +2201,42 @@ def install(self) -> None:
             if ltype == "read":
                 lock.release_read()
                 self._requeue_task(task, install_status)
-                continue
+                return None
 
             # Proceed with the installation since we have an exclusive write
             # lock on the package.
-            install_status.set_term_title(f"Installing {pkg.name}")
+            # Start a child process for a task that's ready to be installed.
+
+            install_status.set_term_title(f"Installing {task.pkg.name}")
```

Comment:
These are going to get called in rapid succession during the start loop and quickly overwrite each other in the terminal title.  Let's sync up on how the status update should work.  I think:
* You should update "installing" status within the main loop below
* Basically look at `active_tasks` there and print the names of all packages being installed.
* The only one messages I would keep in here are the attempts to get locks (which may hang -- so they're useful).